index.html

```diff
@@ -61,5 +60,141 @@ <h1 class="section-title">I’m Polina Glad to see you!</h1>
         </ul>
       </section>
     </main>
+    <section class="cards-project">
+      <div class="card">
```

Comment:
- [ ] **For listing projects, please use a `ul` tag to group them, and `li` per project.**

The `ul` and `li` tag order the element semantically making it obvious to screen readers or assistive technologies to know they belong to the same group. This helps in proper accessibility and optimization of the webpage.

Reply:
Solved